docs/translations.md

```diff
@@ -1,3 +1,53 @@
+# Managing strings
+
+We use Android's [resource system](https://developer.android.com/guide/topics/resources/localization) to localize
+user-visible strings in our apps.
+
+Our source language is English (American English to be more precise, but simply "English" (en) on Weblate).
+
+Feature modules are expected to store strings exclusively used by them in their `res/values/strings.xml` file.
+There's currently no final place for strings shared between feature modules. For now there's some in
+[:legacy:ui:base](/legacy/ui/base/src/main/res/values/strings.xml).
+
+Translations of source strings happen exclusively in our
+[Weblate project](https://hosted.weblate.org/projects/tb-android/). This means the source language is only modified by
+changes to this repository, i.e. via pull requests. Translations are only updated on Weblate and then merged into this
+repository by the Thunderbird team. This is to avoid overlapping changes in both repositories that will lead to merge
+conflicts.
+
+## Adding a string
+
+Add a new string to the appropriate `res/values/strings.xml` file. Be careful when using Android Studio's translations
+editor. It might modify files containing translations. If so, revert those changes before committing the changes.
```

Comment:
This I think we can expand on. Some could consider US English also a "file containing translation". I think you are trying to say that the string should be added to the en-US variant only, and that Android Studio sometimes adds placeholders for all locales if you click the wrong button?

Reply:
To be honest, I don't really know if Android Studio's translations editor is a problem in this case. I rarely use it. I've only seen pull requests from external contributors modify files containing translations. Maybe this should be rewritten to mentioned that nothing but the source language should be changed.